tests/e2e/e2e_ibc_test.go

```diff
@@ -140,7 +140,7 @@ func (s *IntegrationTestSuite) sendIBC(c *chain, valIdx int, sender, recipient,
 		"ibc-transfer",
 		"transfer",
 		"transfer",
-		"channel-0",
+		icaChannelID,
```

Comment:
this is not related to ica, we can rename `icaChannelID` -> `ibcTransferChannelID`